src/renderer/splash.tsx

```diff
@@ -16,81 +20,88 @@ const Splash = memo(() => {
         ipcRenderer.on('checking-port', () => {
             setShowProgressBar(false);
             setOverallProgressPercentage(0.1);
-            setStatus('Checking for available port...');
+            setStatus(t('splash.checkingPort', 'Checking for available port...').toString());
```

Comment:
Let's get rid of the `.toString()`s. 

The problem here is that i18next can allow null values as valid translations. TypeScript chooses a strange overload here which causes `t` to return `string | null`. Luckily, we can configure *the types* of i18next to say: "null is never a valid translation." Add a file `src\i18next.d.ts` with this context:
```ts
// https://www.i18next.com/overview/typescript#create-a-declaration-file
// https://www.i18next.com/overview/typescript#custom-type-options

import 'i18next';

declare module 'i18next' {
    // Extend CustomTypeOptions
    interface CustomTypeOptions {
        returnNull: false;
    }
}
```

and then you should be able to remove all `.toString()`s without any errors.